src/library/Avatar/Avatar.js

```diff
@@ -134,10 +131,14 @@ export default function Avatar({
   const rootProps = {
     icon,
     noBackground,
-    shape,
     size,
     ...restProps
   };
 
   return <Root {...rootProps}>{text || icon || children}</Root>;
 }
+
+Avatar.defaultProps = {
```

Comment:
It is okay to define the defaultProps after the export?  It seems to be working...

Reply:
@kylegach & I were both wondering that. There were a few examples of `displayName` being defined after the export, so I experimented with `defaultProps` & although it seems to work, our inclination is that maybe it's because of a transpile & that may not be safe for our users? So I changed them all back to export after the assignments. 

However, in `Avatar` & `Link`, where `size` & `element` are optional props (respectively, they error when the defaults are defined outside of the declaration. We got around this by accessing the default props, e.g.:

```js
const Link = (props: Props) => {
  const Root = createStyledComponent(
    element || this.defaultProps.element,
    linkStyles,
    ...
  )
  ...
}
Link.defaultProps = {...}
export default Link;
```

This was fine when we had `export default function <Component>`, but Flow errors with the above pattern. For now, I changed those two components back, but we should discuss what the best pattern is & whether the above approach makes sense.

Reply:
Disregard that, @coldpour helped me realize that it was just a matter of `this` changing with the arrow function. Fixed it & now all exports are at the ends of files.

Reply:
Yeah, probably not a big deal either way...since it works regardless...